src/cdk-experimental/menu/menu.md

```diff
@@ -0,0 +1,167 @@
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <button cdkMenuItem [cdkMenuTriggerFor]="submenu">Open Submenu</button>
+  </div>
+</ng-template>
+```
+
+A menuitem also has two sub-types, neither of which should trigger a menu:
+
+- CdkMenuItemCheckbox
+- CdkMenuItemRadio
+
+#### Menuitemcheckbox
+
+A component with the `cdkMenuItemCheckbox` directive behaves just as you would expect a typical checkbox to behave. A use case may be toggling an independent setting since it contains a checked state. A component with the `cdkMenuItemCheckbox` directive does not need the additional `cdkMenuItem` directive.
+
+#### Menuitemradio
+
+A `cdkMenuItemRadio` behaves as you would expect a radio button to behave. A use case may be toggling through more than one state since radio buttons can be grouped together. A component with the `cdkMenuItemRadio` directive does not need the additional `cdkMenuItem` directive.
+
+#### Groups
+
+By default `cdkMenu` acts as a group for `cdkMenuItemRadio` elements. That is, components marked with `cdkMenuItemRadio` added as children of a `cdkMenu` will be logically grouped and only a single component can have the checked state.
+
+If you would like to have unrelated groups of radio buttons within a single menu you should use the `cdkMenuGroup` directive.
+
+```html
+<ng-template cdkMenuPanel #panel="cdkMenuPanel">
+  <div cdkMenu [cdkMenuPanel]="panel">
+    <!-- Font size -->
+    <div cdkMenuGroup>
+      <button cdkMenuItemRadio>Small</button>
+      <button cdkMenuItemRadio>Medium</button>
+      <button cdkMenuItemRadio>Large</button>
+    </div>
+    <hr />
+    <!-- Paragraph alignment -->
+    <div cdkMenuGroup>
+      <button cdkMenuItemRadio>Left</button>
+      <button cdkMenuItemRadio>Center</button>
+      <button cdkMenuItemRadio>Right</button>
+    </div>
+  </div>
+</ng-template>
+```
+
+Note however that when the menu is closed and reopened any state is lost. You must subscribe to the groups `change` output, or to `cdkMenuItemToggled` on each radio item and track changes your self.
+
+<!-- TODO standalone menu example with checkboxes and grouped radios -->
+
+### Accessability
+
+The set of directives defined in `@angular/cdk-experimental/menu` follow accessability best practices as defined in the [aria spec][1]. Specifically, the menus are aware of left-to-right and right-to-left layouts and opened appropriately however it is up to you to handle any related styling. Further, screen readers are supported however you should add `aria-label` or `aria-labelledby` if menu items do not have text. Finally, keyboard interaction is supported as defined in the [aria menubar keyboard interaction spec][2].
+
+<!-- links -->
+
+[1]: https://www.w3.org/TR/wai-aria-practices-1.1/#menu 'Aria Menubar Pattern'
+[2]: https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-12 'Aria Menubar Keyboard Interaction'
```

Comment:
I'm trying to get away from using numbered links now and instead using links with names, e.g.
```suggestion
[menubar]: https://www.w3.org/TR/wai-aria-practices-1.1/#menu 'Aria Menubar Pattern'
[keyboard]: https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-12 'Aria Menubar Keyboard Interaction'
```